core/src/main/resources/hudson/model/AbstractBuild/index.jelly

```diff
@@ -28,13 +28,16 @@ THE SOFTWARE.
   <l:layout title="${it.fullDisplayName}">
     <st:include page="sidepanel.jelly" />
     <l:main-panel>
-      <div style="float:right; z-index: 1; position:relative; margin-left: 1em">
+      <t:buildCaption>${%Build} ${it.displayName} (<i:formatDate value="${it.timestamp.time}" type="both" dateStyle="medium" timeStyle="medium"/>)</t:buildCaption>
+
+      <div style="float:right;z-index: 1; position:relative; margin-left: 1em">
```

Comment:
the page looks a bit funny still with all this floating, but it's mostly a pre-existing issue

Reply:
Yes, maybe we should redesign this whole page so that this header is also visible permanently rather than scrolling out of sight. Github Actions use a much better console log view page...